test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -25,6 +26,20 @@ def test_ec2_pytorch_inference_cpu(pytorch_inference, ec2_connection, region, cp
     ec2_pytorch_inference(pytorch_inference, "cpu", ec2_connection, region)
 
 
+@pytest.mark.model("resnet")
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_CPU_INSTANCE_TYPE, indirect=True)
+@pytest.mark.parametrize("ei_accelerator_type", PT_EC2_EIA_ACCELERATOR_TYPE, indirect=True)
```

Comment:
nit: it's fairly clear from the name, but can you also add `@pytest.mark.integration("elastic_inference")`